src/anshuKJun20/CountMultipleCharacter.java

```diff
@@ -0,0 +1,30 @@
+package anshuKJun20;
+
+public class CountMultipleCharacter {
+   void findcharcter(char[] ch) {
+	   int countdigit =0,countSpecialChar=0, countUpperCase=0,countLowerCase =0,countletters =0;
+	   for(int index = 0; index<ch.length;index++) {
+		   if(Character.isDigit(ch[index])==true) {
+			   countdigit++;
+		   } else if(Character.isLetter(ch[index])){
+			   countletters++;
+			   if(Character.isLowerCase(ch[index])==true) {
+				   countLowerCase++;
+				} if(Character.isUpperCase(ch[index])==true) {
+					countUpperCase++;
```

Comment:
This else if block is wrong. Please think differently here. countletters without using this varibale you can achive expected output.

Reply:
Done
![old](https://user-images.githubusercontent.com/66829451/85881579-740e7400-b7fb-11ea-904b-ebf1c9b2f475.PNG)